ONSdigital/rasrm-ops
# Update for census branch

# Motivation and Context

The ops tool needs has been updated to be used for Census. This has involved removing most of the business specific code. For this request however, Social is being used as the survey type as the Census type hasn't been implemented in some of the other microservices.

This change also introduces the census sample loader code to load the sample.
# What has changed



Social is now the only survey type to use.
The Census sample loader code has been copied in and modified to run within the tool.
Some bits have been removed such as the execute collection exercise button.
Tests have been updated to test the new functionality, while some business specific tests have been removed.
# How to test?



Run make test in this branch and it should pass all tests.
If you run the tool you should be able to create a survey and collection exercise with associated bits (dates, action plans etc). You'll be able to see if the sample loader works successfully by looking at what is put in Redis or what goes on the Case.CaseDelivery rabbitmq queue.
# Links



https://trello.com/c/xfLawjHD/488-census-version-of-the-ops-tool-8
https://github.com/ONSdigital/census-rm-kubernetes/pull/21
